q2_dada2/plugin_setup.py

```diff
@@ -71,14 +71,15 @@
         'n_reads_learn': ('The number of reads to use when training the '
                           'error model. Smaller numbers will result in a '
                           'shorter run time but a less reliable error '
-                          'model.'),
+                          'model. If 0 is provided, all input reads will be '
+                          'used.'),
```

Comment:
`If 0 is provided, all input reads will be used from the first sample.`

https://github.com/qiime2/q2-dada2/pull/36#discussion_r98472656